### PR TITLE
Update coerce functions to parse try to json-encoded strings if we get that from flywheel 0.3 fields

### PR DESCRIPTION
**Pull request by skastel**
@RyanGWU82

### CODE REVIEW

File: flywheel/fields/types.py [hunk 1]
```diff
@@ -310,6 +310,12 @@ def __init__(self):
         super(BoolType, self).__init__()
 
     def coerce(self, value, force):
+        if isinstance(value, basestring):
```
Comment (RyanGWU82) [inline flywheel/fields/types.py]: nice, I didn't know about `basestring`.

**Comment by skastel** [comment]
This should make the majority of 0.4.1 backwards compatible with our systems.

**Comment by RyanGWU82** [comment]
It would probably be prudent to add type checks after loading from JSON.

:+1: otherwise!

**Comment by skastel** [comment]
Type checks happen anyways in the next line

**Comment by RyanGWU82** [comment]
Well, they only happen if `force` is True. I'm not sure of the project's coding style and how often that will be True, but I'm wondering if we should try and be safer when loading from JSON.

**Pull request merged**